lbz/configuration.py

```diff
@@ -1,13 +1,30 @@
+import json
 from abc import ABCMeta, abstractmethod
 from os import getenv
-from typing import Any, Callable, Generic, Optional, TypeVar
+from typing import Any, Callable, Generic, List, Optional, TypeVar, cast
 
 from lbz.aws_ssm import SSM
 from lbz.exceptions import ConfigValueParsingFailed, MissingConfigValue
 
 T = TypeVar("T")
 
 
+class ConfigParser:
+    @staticmethod
+    def split_by_comma(cfg: str) -> List[str]:
+        return cfg.split(",")
+
+    @staticmethod
+    def env_bool(cfg: str) -> bool:
+        if cfg.lower() in ("true", "1"):
+            return True
+        return False
+
+    @staticmethod
+    def load_jwt_keys(cfg: str) -> List[dict]:
+        return cast(list, json.loads(cfg)["keys"])
```

Comment:
Using `cast` is not a silver bullet. First you should think why `mypy` cannot recognize the type and try to help him 😉 

e.g.
```python
deserialized_value: Dict[str, List[dict]] = json.loads(value)
return deserialized_value["keys"]
```

You can even create a custom `TypedDict` instead of declaring it this way - it might be useful in other places.